packages/components/src/components/positions-drawer-card/positions-drawer-card.tsx

```diff
@@ -11,15 +10,63 @@ import {
     getContractTypeDisplay,
     getEndTime,
 } from '@deriv/shared';
+import { TContractInfo } from '@deriv/shared/src/utils/contract/contract-types';
+import type { ContractUpdate } from '@deriv/api-types';
+import moment from 'moment';
+
+type TToastConfig = {
+    key?: number;
+    content: string;
+    timeout?: number;
+    is_bottom?: boolean;
+    type?: string;
+};
+
+type TGetEndTime = Pick<
+    TContractInfo,
+    'is_expired' | 'sell_time' | 'status' | 'tick_count' | 'contract_type' | 'underlying' | 'contract_id'
+> &
+    Required<Pick<TContractInfo, 'date_expiry' | 'exit_tick_time' | 'is_path_dependent'>>;
+
+type TPositionsDrawerCardProps = {
+    addToast: (toast_config: TToastConfig) => void;
+    className?: string;
+    contract_info: TGetEndTime;
+    contract_update: ContractUpdate;
+    currency: string;
+    current_focus: string;
+    display_name: string;
+    getContractById: (contract_id: string) => Record<string, any>;
+    is_mobile: boolean;
+    is_sell_requested: boolean;
+    is_unsupported: boolean;
+    is_link_disabled: boolean;
+    profit_loss: number;
+    onClickCancel: (contract_id: string) => void;
+    onClickSell: (contract_id: string) => void;
+    onClickRemove: () => void;
+    onFooterEntered: () => void;
+    onMouseEnter: () => void;
+    onMouseLeave: () => void;
```

Comment:
@henry-deriv, I think these should be optional, since not every implementation may require definitions for these
 
```suggestion
    onMouseEnter?: () => void;
    onMouseLeave?: () => void;
```